skbio/tree/_upgma.py

```diff
@@ -0,0 +1,90 @@
+# ----------------------------------------------------------------------------
```

Comment:
I thought a bit. Initially I thought that wrappers are trivial and don't have to be a separate module, and WPGMA can be a separate function from UPGMA ("U" stands for "unweighted"). But then I thought that beyond SciPy's implementation, there may more efficient implementations of UPGMA (e.g., [here](https://github.com/DavidCain/upgma)), which we may consider adopt in the future. Being the fastest tree-building method, UPGMA deserves attention (optimization). Therefore, having it in a separate module is justified.